java-agent/src/test/java/io/opentelemetry/test/ClassToInstrument.java

```diff
@@ -1,6 +1,6 @@
 package io.opentelemetry.test;
 
-import io.opentelemetry.auto.api.Trace;
+import io.opentracing.contrib.dropwizard.Trace;
```

Comment:
what happened here?

Reply:
this is to test the trace-annotation instrumentation now that the native `@Trace` annotation is no more